keyboards/kradoindustries/krado66/keymaps/vial/config.h

```diff
@@ -0,0 +1,21 @@
+/* SPDX-License-Identifier: GPL-2.0-or-later */
+
+#pragma once
+
+#define VIAL_KEYBOARD_UID {0xB7, 0x72, 0xC9, 0x62, 0x53, 0xF5, 0xBD, 0xDF}
+#define VIAL_UNLOCK_COMBO_ROWS { 1, 9 }
+#define VIAL_UNLOCK_COMBO_COLS { 0, 7 }
+
+
+#define RGB_DI_PIN B2
+#define RGB_MATRIX_LED_COUNT 14
+#define DRIVER_LED_TOTAL 14
+#define RGBLIGHT_HUE_STEP 8
+#define RGBLIGHT_SAT_STEP 8
+#define RGBLIGHT_VAL_STEP 8
+#define VIALRGB_NO_DIRECT
+
+#define DYNAMIC_KEYMAP_MACRO_COUNT 6
+#define DYNAMIC_KEYMAP_LAYER_COUNT 3
+
+#define VIAL_ENCODER_DEFAULT { KC_PGDN, KC_PGUP, KC_PGDN, KC_PGUP, KC_VOLD, KC_VOLU, RGB_MOD, RGB_RMOD, KC_PGDN, KC_PGUP, KC_PGDN, KC_PGUP, KC_VOLD, KC_VOLU, KC_LEFT, KC_RGHT, KC_LBRC,  KC_RBRC, KC_LBRC, KC_RBRC, C(KC_EQL), C(KC_MINS), KC_WH_U, KC_WH_D}
```

Comment:
Obsolete and redundant settings removed.
```suggestion
#define VIALRGB_NO_DIRECT

#define DYNAMIC_KEYMAP_MACRO_COUNT 6
#define DYNAMIC_KEYMAP_LAYER_COUNT 3
```